zpool.h

```diff
@@ -0,0 +1,40 @@
+
+#include <iostream>
+#include <vector>
+#include <cstring>
+#include <map>
+#include <algorithm>
+
+using namespace std;
+
+static int a = 25;
+class Zpool
```

Comment:
I expect more classes instead of just one class zpool. We can have interface, utility functions etc. into other classes.